packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -157,10 +162,26 @@ static inline BOOL IsHorizontalNavigationWithinZoneAction(FocusZoneAction action
 	return nil;
 }
 
+/// Bypass FocusZone if it's empty or has no focusable elements
+static BOOL ShouldSkipFocusZone(NSView *view)
+{
+    if([view isKindOfClass:[RCTFocusZone class]])
+    {
+        NSView *keyView = GetFirstKeyViewWithin(view);
+        // FocusZone is empty or has no focusable elements
```

Comment:
now this comment is redundant from the function's documentation because you hae improved the getfirstkeyviewwithin()

Reply:
I think it's better to state the possibilities here so that we know all the cases why we want to reject first responder. Will prefer to leave it as is.